src/pizza/commits.service.ts

```diff
@@ -0,0 +1,66 @@
+import { Injectable, NotFoundException } from "@nestjs/common";
+import { InjectRepository } from "@nestjs/typeorm";
+
+import { PageOptionsDto } from "../common/dtos/page-options.dto";
+import { PageDto } from "../common/dtos/page.dto";
+import { Repository } from "typeorm";
+import { PageMetaDto } from "../common/dtos/page-meta.dto";
+import { DbCommits } from "./entities/commits.entity";
+
+@Injectable()
+export class CommitsService {
+  constructor(
+    @InjectRepository(DbCommits, "ApiConnection")
+    private commitsRepository: Repository<DbCommits>
+  ) {}
+
+  baseQueryBuilder() {
+    const builder = this.commitsRepository.createQueryBuilder("commits");
+
+    return builder;
+  }
+
+  async findAllCommitsByBakedRepoId(pageOptionsDto: PageOptionsDto, id: number): Promise<PageDto<DbCommits>> {
+    const queryBuilder = this.baseQueryBuilder();
+
+    queryBuilder.where(`baked_repo_id = ${id}`);
```

Comment:
TypeORM will sanitize the input if we use parameterized queries.

```suggestion
    queryBuilder.where(`baked_repo_id = :id`, { id });
```